lib/pbench/server/database/alembic.ini

```diff
@@ -26,16 +32,25 @@ script_location = alembic
 # versions/ directory
 # sourceless = false
 
-# version location specification; this defaults
+# version location specification; This defaults
 # to alembic/versions.  When using multiple version
-# directories, initial revisions must be specified with --version-path
-# version_locations = %(here)s/bar %(here)s/bat alembic/versions
+# directories, initial revisions must be specified with --version-path.
+# The path separator used here should be the separator specified by "version_path_separator"
```

Comment:
Line 38 looks like it needs wrapping.

Reply:
Agree, but since it is autogenerated, I was thinking I wouldn't touch this.